src/Workspaces/VisualBasic/Portable/FindSymbols/VisualBasicDeclaredSymbolInfoFactoryService.vb

```diff
@@ -291,6 +337,25 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.FindSymbols
             Return False
         End Function
 
+        Private Function IsExtensionMethod(node As MethodBlockSyntax) As Boolean
+            Dim parameterCount = node.SubOrFunctionStatement.ParameterList?.Parameters.Count
+            If Not parameterCount.HasValue Or parameterCount.Value = 0 Then
+                Return False
+            End If
+
+            If TypeOf node.Parent Is ModuleBlockSyntax Then
+                For Each attributeList In node.BlockStatement.AttributeLists
+                    For Each attribute In attributeList.Attributes
+                        If attribute.ArgumentList.Arguments.Count = 0 And attribute.Name.ToString().EndsWith("Extension") Then
```

Comment:
so if they write `<ExtensionAttribute>` then this doesn't work?

Reply:
ah, good point